data_structures/linked-lists.py

```diff
@@ -0,0 +1,53 @@
+class Node():
```

Comment:
This class is great, just a couple notes.

1. This doesn't need to have paren here since you're not passing anything in. Changing it to `class Node:` would be more pythonic in that case.
2. You almost always want to define a `__str__` method on your Node class. more on that in a later comment on your linked list.